core/domain/stats_domain.py

```diff
@@ -641,6 +749,20 @@ def __init__(self, categorized_answer_freq_lists=None):
             if categorized_answer_freq_lists else {})
 
     def to_raw_type(self):
+        """Returns the categorized frequency Python dict.
+
+        Returns:
+            dict. The categorized answer frequency dict of the following format:
+                {
+                    'category': list. The answer frequency list containing dict
```

Comment:
This is wrong. The 'category' string is not hardcoded in the result. It is a variable name.

You will probably want to do something like this:

    dict. A dict whose keys are category names and whose corresponding values are lists of answer frequency dicts. Each answer frequency dict has the following keys and values:
    - 'answer': ...
    - 'frequency': ...

Ditto below.

Reply:
Done!